samples/led-blink/led-blink.csproj

```diff
@@ -2,11 +2,26 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RootNamespace>led_blink</RootNamespace>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Device.Gpio" Version="1.1.0-prerelease.20153.1" />
+  </ItemGroup>
+
+  <!--  
+  If you want to use this project outside of the repo, then delete the following ItemGroup (with the ProjectReference).
+
+  Details:
+  This sample includes references to one or more NuGet packages as both ProjectReference and PackageReference.
+  The PackageReferences are intended for most use cases.
+  The project (as-is) will prefer the ProjectReference. If you want to test the project with the package, type: 
+  dotnet build /p:PreferPackageReference=true
+  The ProjectReference is used by the repo build to test project updates with the samples before they are published 
+  as packages to NuGet.org. This also ensures that the samples don't break.
+  -->
+
+  <ItemGroup Condition="'$(PreferPackageReference)' != 'true'">
```

Comment:
NIT: you could make this condition exclusive by default so people could still copy paste and have it working without the need to delete an ItemGroup. For example:

```xml
<ItemGroup Condition="'$(ForceProjectReference)' != 'true'">
  <PackageReference Include="...
</ItemGroup>

<ItemGroup Condition="'$(ForceProjectReference)' == 'true'">
  <ProjectReference Include="...
</ItemGroup>
```

This way just setting that property to true in the samples/Directory.Build.props will cause the ProjectReference to be used, but if the project gets copied outside of this repo, it just flips to use PackageReference instead without people having to edit it.